src/core/settings.py

```diff
@@ -145,6 +145,7 @@
 
 DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
 STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+SETTINGS_DIR = os.path.join('files', 'media', 'settings')
```

Comment:
What is `SETTINGS_DIR` for?

Reply:
I've changed the name to be hopefully a little more clear.